app/assets/stylesheets/application.scss

```diff
@@ -273,6 +273,7 @@ table {
 
 .table-fixed {
   table-layout: fixed;
+  margin: 0 !important;
```

Comment:
This shouldn't be part of `.table-fixed` - this class is for forcing column widths which are required for clipping, adding a margin here is unexpected behaviour and might have side effects on other pages that use this class.